Automattic/wp-calypso
# WooCommerce.com login flow UI update

#### Proposed Changes

Update the WooCommerce.com login flow to match with the current WooCommerce branding. 

#### Testing Instructions



- On WooCommerce.test click login
- Replace _https://wordpress.com/_ with _http://calypso.localhost:3000/_ in the url
- Go through the login flow
- Go through the registration flow
- Go through the _Email me a login link flow_
- Confirm that all flows match the screenshots bellow (ignore the outdated authorize screen)



#### Screenshots
![Screenshot 2022-07-12 at 10-19-46 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452232-15ab0132-d281-4dc4-94dd-7c898fe03e25.png)
![Screenshot 2022-07-12 at 10-20-14 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452241-255d693b-3da9-4669-ae47-4ad3196518e3.png)
![Screenshot 2022-07-12 at 10-20-45 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452246-1a18bde2-52dd-4f29-b623-31f64b6915ba.png)
![Screenshot 2022-07-12 at 10-21-06 Create a site — WordPress com](https://user-images.githubusercontent.com/1199991/178452250-57e9f487-c3c9-4e3a-8fb1-cd340e8e4aa9.png)
![Screenshot 2022-07-12 at 10-21-42 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452255-9c842a20-b70c-4271-81aa-c1af79f02985.png)
![Screenshot 2022-07-12 at 10-21-53 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452263-9010f165-1eb7-47d2-820b-5bd7cf873542.png)
![Screenshot 2022-07-12 at 10-22-47 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452267-65f98643-0e1f-4e59-af50-3369ba200ac2.png)
![Screenshot 2022-07-12 at 10-23-00 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452269-7bde6e3c-1705-424c-84d8-66016f2dc70b.png)
![Screenshot 2022-07-12 at 10-23-18 Create a site — WordPress com](https://user-images.githubusercontent.com/1199991/178452272-e4333a0d-ad6d-4ed0-acf2-01ebfe377320.png)
![Screenshot 2022-07-12 at 10-23-31 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452275-e4381e2d-7262-4034-a681-a20c566f3b98.png)
![Screenshot 2022-07-12 at 10-23-47 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452279-a4e6dadb-bdd2-4452-85b6-615e9be1a095.png)
![Screenshot 2022-07-12 at 10-24-14 Log In — WordPress com](https://user-images.githubusercontent.com/1199991/178452282-9f730493-f480-4198-bd1e-5ee5e2b6308e.png)
![Screenshot 2022-07-12 at 10-26-17 Authorize WooCommerce com (dev)](https://user-images.githubusercontent.com/1199991/178452284-48dbdcb7-7dd4-440b-865a-4ea6f8429a81.png)
![Screenshot 2022-07-12 at 10-26-28 Authorize WooCommerce com (dev)](https://user-images.githubusercontent.com/1199991/178452288-a2215b79-dfe7-41e4-857c-0ef33c09b701.png)
